util-jvm/src/main/scala/com/twitter/jvm/Hotspot.scala

```diff
@@ -159,17 +166,10 @@ class Hotspot extends Jvm {
   private val log = Logger.getLogger(getClass.getName)
 
   private[this] val safepointBean = {
-    val runtimeBean = Class
-      .forName("sun.management.ManagementFactoryHelper")
-      .getMethod("getHotspotRuntimeMBean")
-      .invoke(null)
+    val runtimeBean = sun.management.ManagementFactoryHelper.getHotspotRuntimeMBean
 
     def asSafepointBean(x: AnyRef) = {
-      x.asInstanceOf[{
-        def getSafepointSyncTime: Long
-        def getTotalSafepointTime: Long
-        def getSafepointCount: Long
-      }]
+      x.asInstanceOf[sun.management.HotspotRuntimeMBean]
```

Comment:
NB: I changed this because the original cast to an anonymous trait depended on reflection and caused a similar warning. This interface has the same methods but also contains an additional method `List<Counter> getInternalRuntimeCOunters()` If we want to hide this we might need a wrapper class.